CHANGELOG.md

```diff
@@ -3,6 +3,40 @@
 ## Current Develop Branch
 - [#9612](https://github.com/MetaMask/metamask-extension/pull/9612): Update main-quote-summary designs/styles
 
+## 8.1.4 Tue Nov 10 2020
```

Comment:
This ain't right

Reply:
All of the dates in this changelog are wrong :sweat_smile:  The date is generated when the changelog is updated.

I think we should update the changelog script to omit a date altogether. Then maybe add it manually in this sync PR. I'll leave that for another day - the real dates can be applied retroactively by looking at the GitHub releases at least.

Reply:
Oh and once we automate our release process, this step can be automated as well.